### Ремизов Дмитрий Владимирович

##### Задача 1

Известно, что генеральная совокупность распределена нормально 
со средним квадратическим отклонением, равным 16. 
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, 
если выборочная средняя M = 80, а объем выборки n = 256.

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

2Ф (t\_lambda) = lambda_

In [3]:
lambda_ = 0.95

Ф (t\_lambda) = 0.475

In [4]:
t_lambda = 1.96

In [5]:
delta = 1.96 * 16 / 16

In [6]:
delta

1.96

Доверительный интервал:

(80 - delta, 80 + delta)

In [7]:
80 - delta

78.04

In [8]:
80 + delta

81.96

Ответ: (78.04, 81.96).

Таблица значений функции Лапласа:

<img src='https://mathprofi.net/files/u/laplas.png'></img>

##### Задача 2

In [9]:
import numpy as np

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, 
получены опытные данные:

6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1

Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, 
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это 
значение с доверительной вероятностью 0,95.

In [10]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
a

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [11]:
a_mean = a.mean()
a_mean

6.590000000000001

In [12]:
a_std = (np.sum((a - a_mean)**2) / 10)**0.5
a_std

0.42766809560686203

In [13]:
a_std = (np.sum((a - a_mean)**2) / (10-1))**0.5
a_std

0.4508017549014448

In [14]:
a.std()

0.42766809560686203

In [15]:
a_std = a.std(ddof=1)
a_std

0.4508017549014448

In [16]:
t_kr = 2.262

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

In [17]:
delta = t_kr * a_std / (10**0.5)

In [18]:
delta

0.32246174408757394

In [19]:
a_mean - delta

6.267538255912426

In [20]:
a_mean + delta

6.912461744087575

ДИ = (6.268, 6.912)

<img src='http://900igr.net/up/datas/109946/035.jpg'>

Таблица критических точек t-распределения Стьюдента

</img>

##### Задача 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. 
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр 
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

H0: M = 17

H1: M > 17

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

Выведем нахождение значение критерия из предыдущей формулы (наблюдаемый критерий обозначим как U_nabl)

U_nabl = delta \* (n\*\*0.5) / sigma

In [21]:
U_nabl = (17.5 - 17) * (100**0.5) / 2
U_nabl

2.5

По таблице функции Лапласа найдем критическую точку для односторонней критической
области:

In [22]:
alpha = 0.05

In [23]:
Ф = (1-2*alpha) / 2

In [24]:
Ф

0.45

In [25]:
u_kr = 1.645

In [35]:
U_nabl < u_kr

False

Ответ: не принимаем H0.

##### Задача 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г. 
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:

202, 203, 199, 197, 195, 201, 200, 204, 194, 190.

Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [30]:
p_cookies = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [31]:
p_cookies_mean = p_cookies.mean()
p_cookies_mean

198.5

In [32]:
p_cookies_std = (np.sum((p_cookies - p_cookies_mean)**2) / (10-1))**0.5
p_cookies_std

4.453463071962462

Введем нулевую гипотезу


H0 : 
    a_mean = 200

при конкурирующей гипотезе 

H1 : 
    a_mean != 200

Вычисляем наблюдаемое значение критерия

In [34]:
Tnabl=(p_cookies_mean-200)*(10**0.5)/p_cookies_std
Tnabl

-1.0651074037450896

По таблице критических точек распределения Стьюдента найдем критическую точку по
уровню значимости a = 0,01 


и числу степеней свободы k = 9 , откуда tкр=3,25.


Так как |Tнабл|=1.065<3,25=tкр, то нулевую гипотезу о равенстве среднего веса 200 г
можно принять.